src/components/Dashboard/TodoList.vue

```diff
@@ -26,8 +26,8 @@
       @okay="resetCancelPaymentErrors()"
       attach="#todo-list"
     />
-
-    <v-expansion-panels v-if="todoItems && todoItems.length > 0" accordion  v-model="panel">
+    <ActionRequired v-if="isStaffActionRequired"/>
```

Comment:
I see that you're conditionally showing this new expansion panel.

However, I believe the API is providing a "todo task" when a business is out of compliance. Could you use that instead?

You can check with Argus on the details of the todo task item.

You can also see that the Todo List component is not handling this task (it should, even if it deliberately ignores it):

![image](https://user-images.githubusercontent.com/10002889/170587821-2f6a9cd3-d5bc-4a9b-8dc4-dbc0e39a392e.png)

Reply:
Faiz and I discussed this and agreed to go with this "special case" implementation for now due to time constraints.

However, we should refactor this some time to fit into the existing "todo" item functionality.

Perhaps when the staff version of this is implemented, this can be refactored accordingly.